app/controllers/users_controller.rb

```diff
@@ -304,15 +304,12 @@ def handle_account_tab
     community_name = ApplicationConfig["COMMUNITY_NAME"]
     @email_body = <<~HEREDOC
       Hello #{community_name} Team,
-      %0A
-      %0A
+      \n
       I would like to delete my account.
-      %0A%0A
```

Comment:
needed to change this because we pass through an ERB::Util.url_encod encoded body that will work on SMTP

Reply:
Internal workings of mail_to